terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/runners.ts

```diff
@@ -352,6 +361,7 @@ async function addSSMParameterRunnerConfig(
 
 function getLaunchTemplateName(runnerParameters: RunnerInputParameters): Array<string | undefined> {
   if (runnerParameters.runnerType.os === 'linux') {
+    /* istanbul ignore next */
```

Comment:
ignore next line code or branch (if there is conditional statements in the next line for example) against code coverage tests. It is a decision in the format of: next statement is not fully tested, mostly because we don't care for a particular side effect or  anything, so don't include in the test coverage reports because we don't care.